tensorboard/plugins/debugger/debugger_plugin_loader.py

```diff
@@ -116,17 +113,26 @@ def load(self, context):
             '\n\nTo use the debugger plugin, you need to have '
             'gRPC installed:\n  pip install grpcio')
       six.reraise(e_type, e_value, e_traceback)
```

Comment:
Now that the import check isn't guarded by the flag, won't this always fail when `tensorflow` isn't present?  Or `grpcio` in principle, though that's now a `tensorboard` pip package dependency so we could probably skip that check these days.

For that reason I'd expect that this PR would now fail the no-TF startup test on Travis.

Reply:
Good point. I moved these import checks under an `if` condition.